charts/fluentd/templates/fluentd-configmap.yaml

```diff
@@ -175,24 +175,48 @@ data:
 
     # Send off to elasticsearch
     <match airflow.**>
-      @id elasticsearch
-      @type elasticsearch_dynamic
-      @log_level info
-      include_timestamp true
-      host "#{ENV['OUTPUT_HOST']}"
-      port "#{ENV['OUTPUT_PORT']}"
-      index_name fluentd.${record["release"]}.${Time.at(time).getutc.strftime(@logstash_dateformat)}
-      <buffer>
-        @type file
-        path "/var/log/fluentd-buffers/#{ENV['RELEASE']}-kubernetes.system.buffer"
-        flush_mode interval
-        retry_type exponential_backoff
-        flush_thread_count 2
-        flush_interval 5s
-        retry_forever
-        retry_max_interval 30
-        chunk_limit_size "#{ENV['OUTPUT_BUFFER_CHUNK_LIMIT']}"
-        queue_limit_length "#{ENV['OUTPUT_BUFFER_QUEUE_LIMIT']}"
-        overflow_action block
-      </buffer>
+      @type copy
+      <store>
+        @type elasticsearch_dynamic
+        @id elasticsearch
+        @log_level info
+        include_timestamp true
+        host "#{ENV['OUTPUT_HOST']}"
+        port "#{ENV['OUTPUT_PORT']}"
+        index_name fluentd.${record["release"]}.${Time.at(time).getutc.strftime(@logstash_dateformat)}
+        <buffer>
+          @type file
+          path "/var/log/fluentd-buffers/#{ENV['RELEASE']}-kubernetes.system.buffer"
+          flush_mode interval
+          retry_type exponential_backoff
+          flush_thread_count 2
+          flush_interval 5s
+          retry_forever
+          retry_max_interval 30
+          chunk_limit_size "#{ENV['OUTPUT_BUFFER_CHUNK_LIMIT']}"
+          queue_limit_length "#{ENV['OUTPUT_BUFFER_QUEUE_LIMIT']}"
+          overflow_action block
+        </buffer>
+      </store>
```

Comment:
can you give me a quick explanation for this part that is applied regardless of the configuration for s3? I would like a comparison to the existing behavior.

Reply:
Here we're using the fluentd's `copy` output plugin to send to multiple destinations. More info here https://docs.fluentd.org/output/copy